utility-emissions-channel/typescript_app/scripts/vault-dev-setup.ts

```diff
@@ -0,0 +1,68 @@
+import Vault from 'node-vault';
+import { readFileSync, writeFileSync } from 'fs';
+import { EOL } from 'os';
+import { join } from 'path';
```

Comment:
@brioux , are you using this file in your setup? if not please delete it.

Reply:
@Zzocker nope, will remove missed this when merging from old branch